There are a couple of TS errors noticed by the Unit tests:

```
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9550_9550&logFilter=debug&logView=flowAware)  Task type_check_client failed after 50983ms.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9551_9551&logFilter=debug&logView=flowAware)       Output for type_check_client
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9552_9552&logFilter=debug&logView=flowAware)    If no output is shown, look for it under the test section for this task.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9553_9553&logFilter=debug&logView=flowAware)    ....STDOUT....
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9554_9554&logFilter=debug&logView=flowAware)    client/my-sites/subscribers/components/add-subscribers-modal/add-subscribers-modal.tsx(43,10): error TS2322: Type 'ExistingReactNode' is not assignable to type 'string'.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9555_9555&logFilter=debug&logView=flowAware)      Type 'number' is not assignable to type 'string'.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9556_9556&logFilter=debug&logView=flowAware)    client/my-sites/subscribers/components/subscribers-page/subscribers-page-context.tsx(54,55): error TS2322: Type 'number | undefined' is not assignable to type 'number | null'.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9557_9557&logFilter=debug&logView=flowAware)    client/my-sites/subscribers/components/subscribers-page/subscribers-page-context.tsx(61,3): error TS2322: Type 'number | undefined' is not assignable to type 'number | null'.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9558_9558&logFilter=debug&logView=flowAware)    client/my-sites/subscribers/hooks/use-unsubscribe-modal.ts(19,50): error TS2345: Argument of type 'number | undefined' is not assignable to parameter of type 'number | null'.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9559_9559&logFilter=debug&logView=flowAware)      Type 'undefined' is not assignable to type 'number | null'.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9560_9560&logFilter=debug&logView=flowAware)    client/my-sites/subscribers/subscriber-details-page.tsx(62,27): error TS2345: Argument of type 'number | null' is not assignable to parameter of type 'number | undefined'.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9561_9561&logFilter=debug&logView=flowAware)    client/my-sites/subscribers/subscriber-details-page.tsx(77,35): error TS2345: Argument of type 'string | null' is not assignable to parameter of type 'string | undefined'.
[13:55:22 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/10339658?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=10339658_9562_9562&logFilter=debug&logView=flowAware)      Type 'null' is not assignable to type 'string | undefined'.
```

--- a/client/my-sites/subscribers/components/add-subscribers-modal/add-subscribers-modal.tsx
+++ b/client/my-sites/subscribers/components/add-subscribers-modal/add-subscribers-modal.tsx
@@ -40,7 +40,11 @@ const AddSubscribersModal = ( {
 	}
 
 	return (
-		<Modal title={ modalTitle } onRequestClose={ onClose } overlayClassName="add-subscribers-modal">
+		<Modal
+			title={ modalTitle as string }
+			onRequestClose={ onClose }
+			overlayClassName="add-subscribers-modal"
+		>
 			{ isUploading && (
 				<>
 					<LoadingBar progress={ 0.5 } />

--- a/client/my-sites/subscribers/components/add-subscribers-modal/add-subscribers-modal.tsx
+++ b/client/my-sites/subscribers/components/add-subscribers-modal/add-subscribers-modal.tsx
@@ -55,7 +55,7 @@ const AddSubscribersModal = ( {
 			) }
 
 			{ ! isUploading && (
-				<label className="add-subscribers-modal__label">{ translate( 'Email or username' ) }</label>
+				<label className="add-subscribers-modal__label">{ translate( 'Email' ) }</label>
 			) }
 
 			<AddSubscriberForm

--- a/client/my-sites/subscribers/components/add-subscribers-modal/style.scss
+++ b/client/my-sites/subscribers/components/add-subscribers-modal/style.scss
@@ -69,7 +69,6 @@
 	}
 
 	.add-subscribers-modal__form--hidden {
-		// display: none;
 		opacity: 0;
 		height: 0;
 		overflow: hidden;

--- a/client/my-sites/subscribers/helpers/index.ts
+++ b/client/my-sites/subscribers/helpers/index.ts
@@ -6,7 +6,7 @@ const getEarnPaymentsPageUrl = ( siteSlug: string | null ) =>
 	`${ URL_PREFIX }/earn/payments/${ siteSlug ?? '' }`;
 
 const getSubscribersCacheKey = (
-	siteId: number | null,
+	siteId: number | undefined | null,
 	currentPage?: number,
 	perPage?: number,
 	search?: string,

--- a/client/my-sites/subscribers/helpers/index.ts
+++ b/client/my-sites/subscribers/helpers/index.ts
@@ -33,7 +33,7 @@ const getSubscribersCacheKey = (
 };
 
 const getSubscriberDetailsUrl = (
-	siteSlug: string | undefined,
+	siteSlug: string | undefined | null,
 	subscriptionId: number | undefined,
 	userId: number | undefined,
 	pageNumber = 1
@@ -46,7 +46,7 @@ const getSubscriberDetailsUrl = (
 };
 
 const getSubscriberDetailsCacheKey = (
-	siteId: number | null,
+	siteId: number | undefined | null,
 	subscriptionId: number | undefined,
 	userId: number | undefined,
 	type: string

--- a/client/my-sites/subscribers/hooks/use-unsubscribe-modal.ts
+++ b/client/my-sites/subscribers/hooks/use-unsubscribe-modal.ts
@@ -8,7 +8,7 @@ import { useRecordRemoveModal } from '../tracks';
 import { Subscriber } from '../types';
 
 const useUnsubscribeModal = (
-	siteId: number | undefined,
+	siteId: number | undefined | null,
 	pageNumber = 1,
 	detailsView = false,
 	onSuccess?: () => void

--- a/client/my-sites/subscribers/mutations/use-subscriber-remove-mutation.ts
+++ b/client/my-sites/subscribers/mutations/use-subscriber-remove-mutation.ts
@@ -9,7 +9,7 @@ import { useRecordSubscriberRemoved } from '../tracks';
 import type { SubscriberEndpointResponse, Subscriber } from '../types';
 
 const useSubscriberRemoveMutation = (
-	siteId: number | null,
+	siteId: number | undefined | null,
 	currentPage: number,
 	invalidateDetailsCache = false
 ) => {

--- a/client/my-sites/subscribers/queries/use-subscribers-query.tsx
+++ b/client/my-sites/subscribers/queries/use-subscribers-query.tsx
@@ -5,7 +5,7 @@ import { getSubscribersCacheKey } from '../helpers';
 import type { SubscriberEndpointResponse } from '../types';
 
 type SubscriberQueryParams = {
-	siteId: number | null;
+	siteId: number | undefined | null;
 	page?: number;
 	perPage?: number;
 	search?: string;

